tests/zfs-tests/include/libtest.shlib

```diff
@@ -37,7 +37,7 @@
 
 function is_linux
 {
-	if [[ $($UNAME -o) == "GNU/Linux" ]]; then
+    if [[ $($UNAME -o) == "GNU/Linux" ]]; then
```

Comment:
I am actually not trying to change this here we decided to spit out into a different pull request